.yamato/urp_lighting-win-dx12.yml

```diff
@@ -140,9 +140,10 @@ URP_Lighting_Win_DX12_playmode_cache_mono_Linear_trunk:
 URP_Lighting_Win_DX12_editmode_mono_Linear_trunk:
     name: URP_Lighting on Win_DX12_editmode_mono_Linear on version trunk
     agent:
-        type: Unity::VM
+        type: Unity::VM::GPU
         image: sdet/gamecode_win10:stable
         flavor: b1.large
```

Comment:
you need to use b1.xlarge to be able to use the rtx2080s, can you please revert these changes? thanks

Reply:
will do

Reply:
done in 264ff4ed6d52ddf676be573920d2311e81bf5627